vault/resource_approle_auth_backend_role_secret_id_test.go

```diff
@@ -58,6 +59,30 @@ func TestAccAppRoleAuthBackendRoleSecretID_wrapped(t *testing.T) {
 	})
 }
 
+func TestAccAppRoleAuthBackendRoleSecretID_wrapped_reuse(t *testing.T) {
+	backend := acctest.RandomWithPrefix("approle")
+	role := acctest.RandomWithPrefix("test-role")
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testutil.TestAccPreCheck(t) },
+		Providers:    testProviders,
+		CheckDestroy: testAccCheckAppRoleAuthBackendRoleSecretIDDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccAppRoleAuthBackendRoleSecretIDConfig_wrapped_reuse(backend, role),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr(secretIDResource, "backend", backend),
```

Comment:
Would probably be a good idea to check the value of the field being added.

Reply:
For the code snippet you've linked this to, I believe from the description that TestCheckResourceAttr compares the data in the secretIDResource with the key 'backend' against the value of backend.. 

i.e. I believe this is already being tested.

For the other attributes, 'wrapping_accessor', 'wrapping_token' and 'accessor', I'm not sure how to check the value since it's random generated by Vault.

Have I misunderstood something here ?